exercises/phone-number/src/test/kotlin/PhoneNumberTest.kt

```diff
@@ -32,21 +33,19 @@ class PhoneNumberTest {
     }
 
     @Ignore
-    @Test
+    @Test(expected = IllegalArgumentException::class)
     fun invalidWhenOnly11Digits() {
-        val expectedNumber = "0000000000"
         val actualNumber = PhoneNumber("21234567890").number
```

Comment:
We should just call the constructor without the `.number` it is unclear where the exception is being thrown (it should be done at object creation as it is currently).